src/server/game.ts

```diff
@@ -85,7 +85,7 @@ class Player {
   constructor(playerName? : string, playerSocketId?: any ) {
     // this.playerId = this.msgId = Math.random() * 10000000000000000;
     this.playerName = playerName || 'Guest';
-    this.playerSocketId = playerSocketId;
+    this.playerSocketId = playerSocketId || undefined;
```

Comment:
I might need to talk to you about this, I need to find a way to maintain a connection to the socket stuff for the game state.